avb/Android.mk

```diff
@@ -51,6 +51,10 @@ LOCAL_EXPORT_C_INCLUDE_DIRS := $(LOCAL_PATH)
 #LOCAL_CLANG := true
 LOCAL_CFLAGS := $(avb_common_cflags) -DAVB_COMPILATION -DUSE_AVB -Wno-error
 
+ifeq ($(BOARD_AVB_ENABLE),true)
+    LOCAL_CFLAGS += -DAVB_AB_I_UNDERSTAND_LIBAVB_AB_IS_DEPRECATED
+endif
+
```

Comment:
Shouldn't this be in  a dedicated patch ?

Reply:
Need not.
It is added because of modify the avb/libavb/uefi_avb_ops.c and cause compile failed without this modification.